tdigest/merging_digest.go

```diff
@@ -481,3 +482,32 @@ func (td *MergingDigest) Data() *MergingDigestData {
 		ReciprocalSum: td.reciprocalSum,
 	}
 }
+
+// decay decays the histo to make values at the top less interesting over time
+func (td *MergingDigest) decay(p float64) {
+	td.mergeAllTemps()
+	for _, c := range td.mainCentroids {
+		c.Weight = c.Weight * p
+		c.Mean = c.Mean * p
```

Comment:
I think we only want to decay the Weight. 
If we decay the Mean, then the digest would also shift towards smaller values, which doesn't seem correct

Reply:
Talk to joe. This was the decision.  We need a way to decay exceptional values over time.